packages/babel-cli/src/babel/file.ts

```diff
@@ -1,5 +1,7 @@
 import convertSourceMap from "convert-source-map";
 import { AnyMap, encodedMap } from "@jridgewell/trace-mapping";
+import type { Section } from "@jridgewell/trace-mapping/dist/types/types";
```

Comment:
I am not comfortable with the undeclared package sub imports. @jridgewell Can you make the `Section` type public?

Reply:
Sure. Does this serve the need for now?

```ts
import type { SectionedSourceMap } from '@jridgewell/trace-mapping';
type Section = SectionedSourceMap['sections'][0]
```

Reply:
Yes, Thanks!